models/repo_list.go

```diff
@@ -308,11 +321,15 @@ func accessibleRepositoryCondition(userID int64) builder.Cond {
 				builder.NotIn("`repository`.owner_id", builder.Select("id").From("`user`").Where(builder.Eq{"visibility": structs.VisibleTypePrivate}))),
 		),
 		// 2. Be able to see all repositories that we have access to
-		builder.In("`repository`.id", builder.Select("repo_id").
-			From("`access`").
-			Where(builder.And(
-				builder.Eq{"user_id": userID},
-				builder.Gt{"mode": int(AccessModeNone)}))),
+		builder.Or(
+			builder.In("`repository`.id", builder.Select("repo_id").
+				From("`access`").
+				Where(builder.And(
+					builder.Eq{"user_id": userID},
+					builder.Gt{"mode": int(AccessModeNone)}))),
+			builder.In("`repository`.id", builder.Select("id").
+				From("`repository`").
+				Where(builder.Eq{"owner_id": userID}))),
```

Comment:
So this was a bug? I see that functions like `LFSObjectAccessible` or even `SearchRepository` relied on this. Shouldn't it be in another PR for a quick fix?

Reply:
I was hoping for some input from someone who knows the ORM a little better than myself, perhaps I queried incorrectly when testing.

Reply:
It looks like a bug, alright. It should be easily testable (I mean, in a running instance).

Reply:
`A repository template can be used to generate a repository based on this one.`

I am open to other wording, I'm not sure I 100% like this wording either.